views/01UCS_LAL-PRIMO_REDESIGN/css/ucla/account.css

```diff
@@ -67,3 +67,30 @@ md-list-item p.text-headings {
   color: var(--color-black);
   font-size: 20px !important;
 }
+
+/* First column of fine information (labels) */
+prm-fines
+  md-list
+  md-list-item
+  .md-list-item-text
+  .weak-text
+  > p
+  > span:first-child {
+  text-transform: uppercase;
+  font-family: Karbon;
+  font-weight: 700;
+  font-size: 16px;
+}
+
+/* Second column of fine information (the data) */
+prm-fines
+  md-list
+  md-list-item
+  .md-list-item-text
+  .weak-text
+  > p
+  > span:nth-child(2) {
+  font-family: Karbon;
```

Comment:
This should be
```
color: var(--black, #0F0F0F);

/* U/Body/Paragraph */
font-family: Karbon;
font-size: 20px;
font-style: normal;
font-weight: 600;
line-height: 160%; /* 32px */
letter-spacing: 0.2px;
```

Reply:
Corrected to reflect full typography entry value